### DISCUSSION TITLE
Help setting up on synology

### DISCUSSION THREAD

**Discussion by pless84** [plausible/analytics#2321] [Self-Hosted Support]
I'm trying to setup self-hosted Plausible on a Synology DS920+ but I'm struggling with the BASE_URL can anybody help me with this?

**Comment by ruslandoga** [comment]
👋 @pless84 

What's the problem? `BASE_URL` can be almost anything that fits the format `<scheme>://<hostname>:<port>`

**Reply by pless84** [reply]
I get the attached log if it help. Something along the lines of non-existing domain even though I use the NAS domain.
[plausible-analytics1.csv](https://github.com/plausible/analytics/files/9771387/plausible-analytics1.csv)

**Reply by ruslandoga** [reply]
@pless84 pardon my ignorance, but what's a NAS domain? :)

**Reply by ruslandoga** [reply]
The errors seem to mean that `plausible` instance can't connect to postgres, it can't lookup its ip address -> `:nxdomain` error. Do you use `docker-compose.yml` setup?

**Reply by pless84** [reply]
> @pless84 pardon my ignorance, but what's a NAS domain? :)

The domain of our nas setup :)

**Reply by pless84** [reply]
> The errors seem to mean that `plausible` instance can't connect to postgres, it can't lookup its ip address -> `:nxdomain` error. Do you use `docker-compose.yml` setup?

No i installed the containers via the synology package manager, maybe I need to find out how to setup with docker-compose instead

**Reply by ruslandoga** [reply]
> The domain of our nas setup :)

🙈 of course ...

> No i installed the containers via the synology package manager

Interesting, could you link me to their docs? I think maybe a bridge network is missing in that setup. ~~Or maybe [an overlay network](https://docs.docker.com/network/overlay/) is needed.~~ I haven't done anything NAS related with docker ...

**Reply by pless84** [reply]
https://www.synology.com/support/documentation it's running DSM 7

**Reply by ruslandoga** [reply]
The link leads to a list of PDFs. Which one is about installing container images with a package manager? :)
Also how do you run the containers, do you use `docker`? Or something else?

**Reply by ruslandoga** [reply]
According to https://linuxhint.com/docker-compose-with-synology/, it's possible to setup docker compose on synology. I'd suggest going that route.

**Reply by pless84** [reply]
Got it up and running via docker-compose thanks for the help :)
